spec/unit/parser_spec.rb

```diff
@@ -0,0 +1,18 @@
+require "dry/web/parser"
+
+RSpec.describe Dry::Web::Parser do
+  let(:klass) { Dry::Web::Parser }
+  let(:parser) { klass.new(file) }
+  let(:file) { SPEC_ROOT.join("fixtures/test/.env.test") }
+
+  describe "#parse" do
+    subject { parser.parse }
```

Comment:
could you please change this to either use `let(:settings) { parser.parse }` or just do it inside the example block using a local var? I know we're using anonymous subjects in some other gems but it's actually something we should move away from, as it makes reading specs harder